app/common/directives/mode-bar/mode-bar.html

```diff
@@ -5,7 +5,7 @@
         <!-- main menu -->
         <ul class="deployment-menu">
             <li ng-class="{ active: activeMode === 'map' }">
```

Comment:
We might be able to use `ui-sref-active` to replace `activeMode` here

Reply:
YEA!  good catch. I missed a lot of stuff like that. Will change it